core/src/main/c/share/jit/impl/x86.h

```diff
@@ -74,20 +74,20 @@ namespace questdb::x86 {
 
         c.cmp(rhs, INT_NULL);
         c.je(l_null);
-        c.vcvtsi2ss(r, r, rhs);
+        c.cvtsi2ss(r, rhs);
         c.jmp(l_exit);
         c.bind(l_null);
-        c.vmovss(r, NaN);
```

Comment:
Looks like we still have `c.vmovss` and `c.vmovsd` calls in `mem2reg` function. Is it on purpose?

Reply:
hmm, should be fixed in the first commit

Reply:
Sorry, my bad. I wasn't fully awaken when reading the changes.